Microsoft.Toolkit.Uwp.UI.Controls/TextToolbar/Formats/RichText/RichTextFormatter.cs

```diff
@@ -26,6 +29,90 @@ public RichTextFormatter(TextToolbar model)
             ButtonActions = new RichTextButtonActions(this);
         }
 
+        /// <summary>
+        /// Invoked the flyout to change the font size
+        /// </summary>
+        /// <param name="button">The button pressed</param>
+        public void FormatFontSize(ToolbarButton button)
+        {
+            FontSizeButton.IsToggled = true;
+
+            if (fontSizeList == null)
+            {
+                fontSizeList = new ListBox { Margin = new Thickness(0), Padding = new Thickness(0), SelectionMode = SelectionMode.Single };
+
+                var flyoutPresenterStyle = new Style(typeof(FlyoutPresenter));
+                flyoutPresenterStyle.Setters.Add(new Setter(FrameworkElement.MaxHeightProperty, 300));
+                fontSizeFlyout = new Flyout { Content = fontSizeList, FlyoutPresenterStyle = flyoutPresenterStyle };
```

Comment:
@agneszitte-nventive could we add this to the template as an attached flyout?  Then we can reference it as a template part and show it here instead?

Reply:
@michael-hawker Thanks for the good idea :) ! I will jump back on this PR in a few days, that will let me some time to get back into context ;)

Reply:
🦙 Looks like we've made some changes since you last worked on this PR, but I'm not sure what they would have been; may be good to restart on top of latest master and use this as a reference for the XAML Flyout layout.